test/general/tizen_sdk_test.dart

```diff
@@ -244,13 +244,11 @@ void main() {
         .createSync(recursive: true);
 
     final Rootstrap rootstrap = tizenSdk.getFlutterRootstrap(
```

Comment:
Now `getFlutterRootstrap` can simply be named `getRootstrap` since there's no such thing as "Flutter rootstrap".

Reply:
Oops. The name conflicts with the local inline function name `getRootstrap`.

Reply:
Changed the name of the local inline function to getTizenRootstrap().